### ISSUE TITLE
フルコンボ演出 / クリア失敗演出

### ISSUE THREAD

**Issue by AioiLight** [AioiLight/TJAPlayer3#23]

**Comment by Goompaolo9500DNX** [comment]
Good idea to put animations for Clear Failure and Full Combo. I only see "Clear Success" if I cleared a song. I don't see anything if I failed a song. Even if I no-missed the song, I don't see Full Combo. I agree on this issue that is not active for more than a year.
